lib/route.js

```diff
@@ -337,7 +337,7 @@ function buildRouting (options) {
         // any route insertion error created by fastify can be safely ignore
         // because it only duplicate route for head
         if (!context[kRouteByFastify]) {
-          const isDuplicatedRoute = error.message.includes(`Method '${opts.method}' already declared for route '${opts.url}'`)
+          const isDuplicatedRoute = error.message.includes(`Method '${opts.method}' already declared for route`)
```

Comment:
The purpose of `${opts.url}` is to give the reader some sort of clue as to which route is causing the issue. Removing it makes things very ambiguous.

Reply:
This part is only a string comparison to identify that the error thrown from `find-my-way` is the error that says that the route is already declared.
The place where `opts.url` does matter is in the Fastify instance error, right after, as you can see the error shows the user exactly what is the problematic route.